tests/test_optuna_run_config_generator.py

```diff
@@ -295,6 +298,7 @@ def test_create_objective_based_run_config_with_request_rate(self):
         self.assertEqual(model_config.to_dict()["name"], self._test_config_dict["name"])
         self.assertEqual(perf_config["batch-size"], DEFAULT_BATCH_SIZES)
         self.assertEqual(perf_config["request-rate-range"], 2048)
+        self.assertNotIn("concurrency", perf_config)
```

Comment:
This should be:
```self.assertEqual(perf_config["concurrency-range"], None)```

Reply:
modified it locally. didn't update the change due to debugging.